### ISSUE TITLE
DynamicArray with non-copyable types. 

### ISSUE THREAD

**Issue by Panke** [dlang-community/containers#171]
```d
/+dub.sdl:
dependency "emsi_containers" version="~>0.8.0"
+/

import containers;

struct Foo
{
    DynamicArray!int i;
}

struct Bar
{
    DynamicArray!Foo foos;
}

void main()
{ 
	Bar b;  
}
```
I think we can and should make this possible.

**Comment by Panke** [comment]
#172

**Comment by CyberShadow** [comment]
Fixed in #182.

**Comment by Panke** [comment]
Thanks!
